### PR TITLE
"Binary Flags" List Overflowing

### PR DESCRIPTION
**Pull request by NumberC**
This solves issue #93

### CODE REVIEW

**Comment by NumberC** [comment]
Capture

File: _config.yml [hunk 1]
```diff
@@ -21,6 +21,9 @@ url: "https://learn.circuitverse.org" # the base hostname & protocol for your si
 permalink:
 exclude: ["node_modules/", "*.gemspec", "*.gem", "Gemfile", "Gemfile.lock", "package.json", "package-lock.json",  "script/", "LICENSE.txt", "lib/", "bin/", "README.md", "Rakefile", "vendor"]
 
+#gets rid of Netlify error
+repository: NumberC/Interactive-Book
```
Comment (tachyons) [inline _config.yml]: This shouldn't be the part of the PR

File: _includes/fsm.html [hunk 2]
```diff
@@ -1,9 +1,9 @@
-<html>
+﻿<html>
 <head>
 
     <script src="../assets/js/ModuleCheckpoints.js"></script>
     <script
-            src="../assets/js/vendor/jquery-3.4.1.min.js"></script>
+            src="https://ajax.aspnetcdn.com/ajax/jQuery/jquery-1.10.2.min.js"></script>
```
Comment (tachyons) [inline _includes/fsm.html]: why downgrading jquery?

File: _config.yml [hunk 3]
```diff
@@ -21,6 +21,9 @@ url: "https://learn.circuitverse.org" # the base hostname & protocol for your si
 permalink:
 exclude: ["node_modules/", "*.gemspec", "*.gem", "Gemfile", "Gemfile.lock", "package.json", "package-lock.json",  "script/", "LICENSE.txt", "lib/", "bin/", "README.md", "Rakefile", "vendor"]
 
+
+
+
```
Comment (tachyons) [inline _config.yml]: ✂️

File: _config.yml [hunk 4]
```diff
@@ -29,8 +32,7 @@ aux_links:
   "Circuitverse on GitHub":
     - "//github.com/CircuitVerse"
  
-# Repository name for Setting up Netlify
-repository: CircuitVerse/Interactive-Book
+
```
Comment (tachyons) [inline _config.yml]: ✂️

File: _includes/fsm.html [hunk 5]
```diff
@@ -1,4 +1,4 @@
-<html>
+﻿<html>
```
Comment (tachyons) [inline _includes/fsm.html]: 🤔

File: _config.yml (hunk 3 above)
Comment (NumberC) [reply _config.yml]: Sorry, I was working on my phone. It should be fixed now

File: _config.yml (hunk 4 above)
Comment (tachyons) [reply _config.yml]: Remove extra space introduced here

File: _config.yml [hunk 6]
```diff
@@ -28,10 +28,9 @@ search_enabled: true
 aux_links:
   "Circuitverse on GitHub":
     - "//github.com/CircuitVerse"
- 
+
```
Comment (sal2701) [inline _config.yml]: @NumberC give this a look :grimacing:

**Pull request merged**